binding.gyp

```diff
@@ -97,6 +97,10 @@
       "include_dirs": [
         "ffmpeg/ffmpeg-static-build/include"
       ],
+      "library_dirs": [
+        "/opt/homebrew/lib",
+        "/usr/local/lib"
```

Comment:
@Tommy-Hu this section is executed when it's mac os. In your slack message you are referring to CI issue. Or this change is not related?